internal/runnerinstall/ecs.go

```diff
@@ -160,12 +160,26 @@ func (i *ECSRunnerInstaller) Install(ctx context.Context, opts *InstallOpts) err
 				return err
 			}
 
-			if netInfo, err = awsinstallutil.SetupNetworking(ctx, ui, sess, i.Config.Subnets); err != nil {
+			// TODO: Add a port here if the user specified the -liveness-tcp-addr flag
```

Comment:
Does this need to be turned into a GH issue to be tracked?

Reply:
Sure, I'll add one!

Reply:
https://github.com/hashicorp/waypoint/issues/4254